From c8467669b973bcb444d35b4cdfada84e33f282b9 Mon Sep 17 00:00:00 2001
From: cybojenix <anthonydking@gmail.com>
Date: Tue, 10 Dec 2013 20:19:38 +0000
Subject: [PATCH] qcom media-caf: final make file patches for msm7x27a

it's msm7x27a, not msm7627a
msm7x27a doesn't use c2dcolorconvert or mm-video. disable it

Change-Id: I1c7ebd4a97b8eeb84959e85800ae27867050f931
---
 Android.mk                    | 2 +-
 libc2dcolorconvert/Android.mk | 2 ++
 mm-core/Android.mk            | 2 +-
 mm-video/Android.mk           | 2 ++
 4 files changed, 6 insertions(+), 2 deletions(-)

diff --git a/Android.mk b/Android.mk
index 3a1c84b..7064f6b 100644
--- a/Android.mk
+++ b/Android.mk
@@ -1,4 +1,4 @@
-ifneq ($(filter msm8960 msm8660 msm7x30,$(TARGET_BOARD_PLATFORM)),)
+ifneq ($(filter msm8960 msm8660 msm7x30 msm7x27a,$(TARGET_BOARD_PLATFORM)),)
 ifeq ($(TARGET_QCOM_MEDIA_VARIANT),caf)
 
 include $(call all-subdir-makefiles)
diff --git a/libc2dcolorconvert/Android.mk b/libc2dcolorconvert/Android.mk
index 4d774ee..06c2e1d 100644
--- a/libc2dcolorconvert/Android.mk
+++ b/libc2dcolorconvert/Android.mk
@@ -1,3 +1,4 @@
+ifneq ($(TARGET_BOARD_PLATFORM),msm7x27a)
 LOCAL_PATH := $(call my-dir)
 
 include $(CLEAR_VARS)
@@ -26,3 +27,4 @@ LOCAL_MODULE := libc2dcolorconvert
 LOCAL_ADDITIONAL_DEPENDENCIES := $(TARGET_OUT_INTERMEDIATES)/KERNEL_OBJ/usr
 
 include $(BUILD_SHARED_LIBRARY)
+endif
diff --git a/mm-core/Android.mk b/mm-core/Android.mk
index a0c56f9..7dd3431 100644
--- a/mm-core/Android.mk
+++ b/mm-core/Android.mk
@@ -10,7 +10,7 @@ OMXCORE_CFLAGS += -U_ENABLE_QC_MSG_LOG_
 #             Figure out the targets
 #===============================================================================
 
-ifeq ($(TARGET_BOARD_PLATFORM),msm7627a)
+ifeq ($(TARGET_BOARD_PLATFORM),msm7x27a)
 MM_CORE_TARGET = 7627A
 else ifeq ($(TARGET_BOARD_PLATFORM),msm7x30)
 MM_CORE_TARGET = 7630
diff --git a/mm-video/Android.mk b/mm-video/Android.mk
index 2d0a533..8d623a5 100755
--- a/mm-video/Android.mk
+++ b/mm-video/Android.mk
@@ -1,6 +1,8 @@
+ifneq ($(TARGET_BOARD_PLATFORM),msm7x27a)
 OMX_VIDEO_PATH := $(call my-dir)
 include $(CLEAR_VARS)
 
 include $(OMX_VIDEO_PATH)/vidc/vdec/Android.mk
 include $(OMX_VIDEO_PATH)/vidc/venc/Android.mk
 include $(OMX_VIDEO_PATH)/DivxDrmDecrypt/Android.mk
+endif
-- 
1.9.1

